conf/messages

```diff
@@ -115,6 +115,8 @@ features.allowPayments.enabled = Payments allowed
 features.date-service.staticDate.enabled = Use static date (2018-05-01)
 features.allowNineBox.enabled = Nine box page allowed
 features.auditing.enabled = Auditing service enabled
+features.useVatReturnsService.enabled = Use vat-returns microservice enabled
```

Comment:
Can these two messages have either "enabled" taken off the end or "Use" taken off the start. Using both words seems unnecessary.